tas/views.py

```diff
@@ -45,42 +49,63 @@ def get_departments_json(request):
         departments = {}
     return HttpResponse(json.dumps(departments), content_type='application/json')
 
-def _create_ticket_for_pi_request(user):
+def _create_ticket_for_pi_request(user, is_federated=False):
     """
     This is a stop-gap solution for https://collab.tacc.utexas.edu/issues/8327.
     """
     rt = rtUtil.DjangoRt()
     subject = "Chameleon PI Eligibility Request: %s" % user['username']
+    if is_federated:
+        problem_description = "This PI Eligibility request can be reviewed at " +\
```

Comment:
Nice--so we will still send all the eligibility requests to RT, we will just have to update ticketmasters to know that they must be approved in Portal exclusively? Or to both TAS and Portal?

Reply:
As-is a PI Eligibility request made by federated users doesn't make an equivalent request in TAS so those requests would need to be approved in Portal, the text here should let ticketmasters know which requests should be reviewed in portal